screens/Profile.tsx

```diff
@@ -0,0 +1,142 @@
+import { StyleSheet, Image, ScrollView } from 'react-native';
+
+import { FontAwesome } from '@expo/vector-icons';
+import { Text, View } from '../components/Themed';
+import { RootTabScreenProps } from '../types';
+
+export default function TabOneScreen({ navigation }: RootTabScreenProps<'TabOne'>) {
```

Comment:
everything looks good! was able to get it running on my machine. but, I feel like we should rename it away from TabOneScreen to something more descriptive.